stream-chat-android/src/main/java/com/getstream/sdk/chat/ImageLoader.kt

```diff
@@ -41,13 +43,28 @@ public object ImageLoader {
             }
     }
 
-    public fun ImageView.loadWithGifSupport(
+    public fun ImageView.load(@RawRes @DrawableRes drawableResId: Int) {
+        loadAny(drawableResId)
+    }
+
+    public fun ImageView.load(uri: Uri?): Unit = loadAny(uri)
+
+    @JvmStatic
+    @JvmOverloads
+    public fun ImageView.load(
         uri: String?,
-        @DrawableRes placeholderResId: Int?,
+        @DrawableRes placeholderResId: Int? = null,
+        onStart: () -> Unit = {},
+        onComplete: () -> Unit = {},
+    ): Unit = loadAny(uri, placeholderResId, onStart, onComplete)
```

Comment:
Looks like we should be able to merge these two methods into just one

Reply:
I didn't want to expose `loadAny` outside this class. IMO it is better to use type-safe versions

Reply:
Nevermind, didn't realize one of these took a `Uri` and the other a `String` as its parameter 🤦